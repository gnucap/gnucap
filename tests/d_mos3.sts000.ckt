nmos n gate, sub-threshold
Vgs   1  0  0.236734
M1   2  1  0  4  cmosn  l= 9.u  w= 9.u  nrd= 1.  nrs= 1. 
Vds   3  0  5.
Rds   2  3  100.K
Vbs   4  0 -1.234875
.model cmosn  nmos ( level=3  vto= 0.844345  kp= 41.5964u  gamma= 0.863074 
+ phi= 0.6  rd= 0.  rs= 0.  cbd= 0.  cbs= 0.  is= 0.  pb= 0.7  cgso= 218.971p 
+ cgdo= 218.971p  cgbo= 0.  rsh= 0.  cj= 384.4u  mj= 0.4884  cjsw= 527.2p 
+ mjsw= 0.3002  js= 0.  tox= 41.8n  nsub= 15.3142E+15  nss= 1.E+12 
+ nfs= 3.5934E+12  tpg=1  xj= 400.n  ld= 265.073n  uo= 503.521 
+ kf= 0.  af= 1. 
+ fc= 0.5  delta= 0.36745  theta= 0.  eta= 0.  kappa= 0.2 )
*+(* vfb=-0.4241892 )
*    lambda= 0.00906241  vmax= 55.9035K
.options trace
.print op v(nodes) iter(0)
.op
*>.status notime
.end
