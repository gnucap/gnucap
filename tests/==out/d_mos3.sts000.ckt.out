nmos n gate, sub-threshold 
>>>>>.print op v(nodes) iter(0) 
cmosn: rsh - rs - rd conflict: using rsh
cmosn: is - js conflict
>>>>>.op 
cmosn: rsh - rs - rd conflict: using rsh
cmosn: is - js conflict
#           v(1)       v(2)       v(3)       v(4)       iter(0)   
new max = 5, new limit = 5.5
 27.        0.23673    4.9998     5.        -1.2349     5.        
>>>>>*>.status notime 
Gnucap   System status
iterations: op=5, dc=0, tran=0, fourier=0, total=5
transient timesteps: accepted=0, rejected=0, total=0
nodes: user=4, subckt=0, model=0, total=4
dctran density=75.0%, ac density=75.0%
